packages/gestalt/src/Tabs.js

```diff
@@ -53,10 +56,13 @@ export default class Tabs extends React.Component<Props, State> {
   handleTabMouseLeave = () => this.setState({ hoveredTabIndex: undefined });
 
   render() {
-    const { tabs, activeTabIndex } = this.props;
+    const { tabs, activeTabIndex, wrap } = this.props;
     const { focusedTabIndex, hoveredTabIndex } = this.state;
     return (
-      <div className={styles.Tabs} role="tablist">
+      <div
+        className={classnames(styles.Tabs, wrap === true && layout.flexWrap)}
```

Comment:
why not just `wrap && layout.flexWrap` since only a truthy value will hit the second clause and since its typed to a boolean, we know the only truthy value is `true`